docs/_guides/codewind-crw-quick-guide.md

```diff
@@ -0,0 +1,243 @@
+---
+layout: guide
+summary_title: "Codewind in CodeReady Workspaces"
+title: "Getting Started with Codewind in CodeReady Workspaces"
+categories: guides 
+guide_description: "Use Codewind in CodeReady Workspaces to build high quality cloud native applications."
+permalink: codewind-crw-quick-guide.html
+duration: 15 minutes
+tags: Codewind, CodeReady Workspaces, Openshift, Kubernetes, containers, microservice
+--- 
+
+## Objectives
+* Install CodeReady Workspaces and Codewind
+* Develop a simple microservice that uses Eclipse Codewind on CodeReady Workspaces
+
```

Comment:
Delete the Objectives (lines 12, 13, and 14) from the content because they should be in the metadata; see previous comment.